doc/sphinx/source/introduction.rst

```diff
@@ -50,6 +50,80 @@ these challenging objectives the ESMValTool is conceived as a framework
 which allows community contributions to be bound into a coherent
 framework.
 
+.. _contact:
+
+Contact
+=======
+
+See `www.esmvaltool.org <https://www.esmvaltool.org>`_ for general contact information.
+
+.. _mailing-list:
+
+User mailing list
+-----------------
+
+Subscribe to the ESMValTool user mailing list
+`esmvaltool@listserv.dfn.de <mailto:esmvaltool@listserv.dfn.de>`__
+to stay up to date about our the monthly online meetings, upcoming workshops, and trainings.
+
+To subscribe, send an email to
+`sympa@listserv.dfn.de <mailto:sympa@listserv.dfn.de?subject=subscribe%20esmvaltool>`_
+with the following subject line:
+
+-  *subscribe esmvaltool*
+
+or
+
+-  *subscribe esmvaltool YOUR_FIRSTNAME YOUR_LASTNAME*
+
+The mailing list also has a `public archive <https://www.listserv.dfn.de/sympa/arc/esmvaltool>`_ online.
+
+.. _discussions_page:
+
+Discussions page
+----------------
+
+The `ESMValTool Discussions page <https://github.com/ESMValGroup/ESMValTool/discussions>`__
+is open for all general and technical questions on the ESMValTool, for example about installation, application, development, etc.
+
+.. _core-team:
+
+Monthly meetings
+----------------
+
+We have monthly online meetings using `zoom <https://zoom.us/>`__, anyone with
+an interest in the ESMValTool is welcome to join these meetings to connect with
+the community.
+These meetings are always announced in an issue
+on the `ESMValTool <https://github.com/ESMValGroup/ESMValTool/issues>`_
+repository and on the mailing-list_.
+
+Core development team
+---------------------
+
+-  Deutsches Zentrum für Luft- und Raumfahrt (DLR), Institut für Physik
+   der Atmosphäre, Germany (PI)
+
+   ESMValTool Core PI and Developer: contact for requests to use the
+   ESMValTool and for collaboration with the development team, access to
+   the PRIVATE GitHub repository.
```

Comment:
```suggestion
   - ESMValTool Core PI and Developer: contact for requests to use the ESMValTool and for collaboration with the development team, access to the PRIVATE GitHub repository.
```

Reply:
Attempt at making this render as a sublist item, to make it more clear the text belongs to the first item of the list.